README.md

```diff
@@ -47,24 +47,51 @@ Then there are two main ways to extend the test data: either by adding new datas
 * Following [best practice](https://r-pkgs.org/data.html#sec-data-data), each dataset is stored as a `.rda` file whose name is consistent with the name of the dataset, e.g., dataset `xx` is stored as `xx.rda`. The easiest way to achieve this is to use `usethis::use_data(xx)`
 * The programs in `data-raw/` are stored within the `{pharmaversesdtm}` GitHub repository, but they are **not** part of the `{pharmaversesdtm}` package--the `data-raw/` folder is specified in `.Rbuildignore`.
 * When you run a program that is in the `data-raw/` folder, you generate a dataset that is written to the `data/` folder, which will become part of the `{pharmaversesdtm}` package.
-* The names and sources of test datasets are specified in `R/data.R`, for the purpose of generating documentation in the `man/` folder.
+* The names and sources of test datasets are specified in `R/*.R`, for the purpose of generating documentation in the `man/` folder.
+
+**Note:** The documentation process in `{pharmaversesdtm}` is automated for consistency and ease of maintenance. Metadata for each dataset, such as names, labels, descriptions, authors, and sources, is managed in a centralized CSV file (`inst/extdata/sdtms-specs.csv`) and used to generate `.R` documentation files. See the [Documentation Process](#documentation-process) for details.
```

Comment:
Why is this a `.csv` while the one in pharmaverseadam is a `.xlsx`? Can we align?